guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -92,3 +92,27 @@ Default_Organization_Centos_Stream_BaseOS_x86_64_os               BaseOS x86_64
 Uploading Enabled Repositories Report
 ----
 . Check the `/etc/yum.repos.d/redhat.conf` and ensure that the appropriate repositories have been enabled.
+
+==== Install a package
+
+Packages can now be installed while logged in to the Content Host
+
+----
+ # yum install screen
+----
+
+To manage packages through the {ProjectServer}, configure `Remote Execution`.
+For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs] in the  _Managing Hosts_ guide.
+
+Once configured, use this procedure to install a package from the {ProjectServer} web UI.
+
+.Procedure
+. In the {Project} web UI, navigate to *Hosts* > *Content Hosts*.
+. Click the name of the host which was registered above.
+. Click the *Packages* tab.
+. From the dropdown, click *Actions*.
+. Select *Package Install* from the dropdown.
+. Type *screen* into the text field.
+. From the dropdown next to the *Perform* button, select `via remote execution`.
```

Comment:
```suggestion
. From the dropdown next to the *Perform* button, select `via remote execution`.
```
Could you share a screenshot so I know the right UI label element to reference according to the IBM Style Guide :)